src/main/java/org/jitsi_modified/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -236,11 +233,11 @@ else if (incomingBitrateInitialized)
             // enough data point are left within the current window.
             // Reset incoming bitrate estimator so that the window
             // size will only contain new data points.
-            incomingBitrate = new RateStatistics(kBitrateWindowMs, kBitrateScale);
+            incomingBitrate = new BitrateTracker(Duration.ofMillis(kBitrateWindowMs));
             incomingBitrateInitialized = false;
         }
 
-        incomingBitrate.update(payloadSize, arrivalTimeMs);
+        incomingBitrate.update(new DataSize(8L * payloadSize), arrivalTimeMs);
```

Comment:
Hm, do we want to add some static method accessible from Java to `DataSize` to get a datasize of a number of bytes?

Reply:
I pushed a cleaner version. WDYT?